run/events_pubsub/e2e_test.go

```diff
@@ -25,10 +25,9 @@ import (
 )
 
 func TestPubSubEventsService(t *testing.T) {
-	t.Skip("Service is broken: https://github.com/GoogleCloudPlatform/golang-samples/issues/1566")
 	tc := testutil.EndToEndTest(t)
 
-	service := cloudrunci.NewService("pubsub-events", tc.ProjectID)
+	service := cloudrunci.NewService("run_events_pubsub", tc.ProjectID)
```

Comment:
I believe this must be hyphens.

Reply:
I couldn't find that requirement looking at the source and didn't see an error here.

Reply:
Cool, if it deploys and produces a working URL then it's good. I am surprised this is the case.